### PR TITLE
Dev board

### PR DESCRIPTION
**Pull request by StephanGalvez**
lakshay changes: aligned mounting holes and currently fixing test points

### CODE REVIEW

File: Dev_Board/Dev_Board.kicad_pcb [hunk 1]
```diff
@@ -0,0 +1,6563 @@
+(kicad_pcb (version 20221018) (generator pcbnew)
```
Comment (LeftistTachyon) [inline Dev_Board/Dev_Board.kicad_pcb:1]: Make sure you're following JLCPCB's manufacturing capabilities. You can check these on the website, and I've went ahead and added them to your board & made it compliant to their capabilities.

**Review by LeftistTachyon — CHANGES_REQUESTED** [review]
Good work so far! I bet it was difficult getting the physical dimensions down, and there's just a few nitpicks until you're good to go!

File: Dev_Board/Dev_Board.kicad_pcb (hunk 1 above)
[the remark this answers is not in the document]
Comment (StephanGalvez) [reply Dev_Board/Dev_Board.kicad_pcb:1]: fixed em

**Comment by Lakshay983** [comment]
Everything is there and would work, but there are a lot of issues with readability and neatness that will improve board usability and debugging

Here are some of my suggestions for improving the schematic readability:
- Screenshot 2023-11-24 at 5 02 33 PM things like GND and PWR shouldn't be regular labels. There are symbols for them that exist in KiCAD. The GND symbol is just called GND and for power, you can look it up in the symbol menu as well (3.3V or 24V)
- Screenshot 2023-11-24 at 5 12 03 PM Not too big of a fan of the horizontal connectors, I'd suggest flipping them and making them vertical that way you can make sure your labels can always be seen. (C3, B3, and A3 are covered up in the left connector).
- Try labeling what connections are coming in from the Devboard and what connections are being sent out. Check the Turret schematic for a similar example.

Layout stuff:

- Screenshot 2023-11-24 at 5 18 09 PM this test point is still on the bottom layer (indicated by blue) so it won't be accessibly while this board is stacked on top of the dev board
-  Screenshot 2023-11-24 at 5 19 00 PM what happened to the mounting holes???
- For any test points I'd like to see silkscreen labels as to what they're measuring
-  Screenshot 2023-11-24 at 5 21 58 PM 
Screenshot 2023-11-24 at 5 22 15 PM  The "Out_Port" silkscreens are on different sides and not aligned. Please pick one side and ensure that any silkscreen that needs to be read while the board is in use is on the top side (bottom text will not be visible while the dev board is not connected)
- Instead of writing "Out_Port" for connector names please put what their function is. If they're just generic connectors then that's fine, but if it's say a power connector, a can connector, or a signal connector then please label it so. I.e if there's an intended purpose for the connector then put that on the silkscreen
- None of the test points are aligned. For aligning stuff you can select the components, right-click them, "Align/Distribute" -> "Align to Center" (horizontal or vertical center depending on which way you want to align it)

If you have any questions then please let me know! Don't take what I say for fact, you worked on this board not me so there may be some considerations I don't know.

**Pull request closed without merging**